cli/src/main.rs

```diff
@@ -168,7 +168,7 @@ async fn run_test(test_args: TestArgs) -> anyhow::Result<i32> {
         .unwrap_or(run_exit_code);
 
     let exec_start = run_result.exec_start;
-    match run_upload(
+    if let Err(e) = run_upload(
         upload_args,
         Some(command.join(" ")),
         None, // don't re-run quarantine checks
```

Comment:
Should this be `quarantine_run_result` rather than `None`? I see the quarantine check running twice in the current output (once here in `run_test()`, then again in `run_upload()`) presumably due to this conditional? https://github.com/trunk-io/analytics-cli/blob/a0ff6550727e91f91396196590265d398e658df8/cli/src/upload.rs#L148